service/templates/base.html

```diff
@@ -6,7 +6,10 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   {% block assets %}
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <style>
+    @layer foundation, base, project;
+    @import url('https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css') layer(foundation);
```

Comment:
**Explanation.** Instead of using `<link>` to load Bootstrap, I use [@import](https://developer.mozilla.org/en-US/docs/Web/CSS/@import), so it can be in the lowest `layer()`.

<details><summary><strong>Why?</strong></summary>

So Core-Styles can easily overwrite Bootstrap. And "layers" are the most controlled way to overwrite styles. _Reference: [TACC CSS Layers](https://confluence.tacc.utexas.edu/x/b53tDg)._

</details>